.circleci/config.yml

```diff
@@ -1268,6 +1268,15 @@ jobs:
       - store_artifacts:
           path: webui/react/coverage/lcov-report
 
+  test-unit-downstream:
```

Comment:
hmm, would it fit better if it's categorized as an integration test? 

wdy about limiting this to run only on prs opened from upstream. if that's possible

Reply:
> hmm, would it fit better if it's categorized as an integration test?


yeah good call


> wdy about limiting this to run only on prs opened from upstream. if that's possible
i would rather have it run on forks as well, that way it "fails safe".  

if it automatically fails when it is not able to run, then it is easier to make sure that it is run, and we don't need any additional mechanisms (codeowners, checklist, PSAs)

Reply:
wouldn't that mean that there'd be a failing test for all prs originating from forks?

Reply:
can make it so where it skips for things that don't touch shared.

Reply:
perfect

Reply:
ok it's up, will make some test PRs then put them in the description

Reply:
hmm unfortunately it would be hard to test the skip because of how we get what's changed

`MERGE_BASE="$(git merge-base upstream/master HEAD)"`

emphasis on `upstream/master`

Reply:
i will hard code it to `upstream/test-downstream` for testing purposes and switch back before merge.